src/robusta/core/sinks/msteams/msteams_sink_params.py

```diff
@@ -4,6 +4,7 @@
 
 class MsTeamsSinkParams(SinkBaseParams):
     webhook_url: str
+    use_emoji: bool
```

Comment:
I would add a default value:
`use_emoji: bool = False`
Someone can override, otherwise it's False